src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/HMPPSAuthService.kt

```diff
@@ -23,6 +23,7 @@ class UnverifiedEmailException : RuntimeException()
 data class UserDetail(
   override val firstName: String,
   override val email: String,
+  override val lastName: String,
 ) : ContactablePerson
```

Comment:
_observation_
Given the following interface and classes:

```kt
interface ContactablePerson {
  val firstName: String
  val email: String
  val lastName: String
}

data class UserDetail(
  override val firstName: String,
  override val email: String,
  override val lastName: String,
) : ContactablePerson

data class ResponsibleOfficer(
  val firstName: String?,
  val email: String?,
  val staffId: Long?,
  val lastName: String?,
)

data class ResponsibleProbationPractitioner(
  override val firstName: String,
  override val email: String,
  val deliusStaffId: Long?,
  val authUser: AuthUser?,
  override val lastName: String,
) : ContactablePerson
```

Should `ResponsibleOfficer` and `ResponsibleProbationPractitioner` be merged?